Jenkinsfile-smoke.groovy

```diff
@@ -0,0 +1,16 @@
+/*
+ * Requires: https://github.com/RedHatInsights/insights-pipeline-lib
+ */
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib") _
+
+
+if (env.CHANGE_ID) {
+    runSmokeTest (
+        ocDeployerBuilderPath: "catalog/catalog-api",
+        ocDeployerComponentPath: "catalog/catalog-api",
+        ocDeployerServiceSets: "catalog,topological-inventory,sources,approval,platform-mq",
+        iqePlugins: ["iqe-self-service-portal-plugin"],
```

Comment:
@brahmanim Where does `self-service-portal` come from?  It is inconsistent with all the other `catalog` references and should be changed if possible.

Reply:
It's the name of the plugin in the internal test framework. Yeah, it could be changed ... what do you think @brahmanim?

Reply:
Yes, SSP was the name of the Catalog Sas, when we create the test plugin for that.
It could be changed, but for now, it is an internal framework name.
I won't change it now since the Catalog name might also change again.

Reply:
@brahmanim since RBAC isn't in this list, will the deployment point to the QE RBAC pod?  I'm only asking because if not then there will be some issues with the API if the `BYPASS_RBAC` environment variable is not set.

Reply:
We could set that env var for catalog here until we get rbac in e2e-deploy: https://github.com/RedHatInsights/e2e-deploy/blob/master/env/smoke.yml

Reply:
@bsquizz is this smoke.yml file use for testing purpose only?
Since I guess that BYPASS_RBAC environment variable that @syncrou mentioned should be used during the deployment of the catalog-api pod.

Reply:
Yes, the smoke env YAML contains settings used by the apps in a smoke deployment only.

Reply:
@bsquizz & @brahmanim Reading through the comments, it appears that the `BYPASS_RBAC` env variable can be set in a separate PR in the `smoke.yml` discussed above. Should this PR be dependent on the PR if my statement is correct?

Reply:
@syncrou yes your statement is correct!

Reply:
@bsquizz @carbonin from what I saw on the catalog/catalog-api template there is no use of BYPASS_RBAC env parameter at all.
Is that mean that the catalog-api.yaml template should be updated to set this BYPASS_RBAC env parameter to false by default.
And the smoke env YAML can overite this value to set to TRUE.

Reply:
@brahmanim Did you mean to mention @syncrou above?

Also, note that the BYPASS_RBAC is a simple check it the env variable exists or not, the value is not evaluated.  See https://github.com/ManageIQ/catalog-api/blob/606c8c81967382b97cfe3cb6e384df79be04346f/lib/rbac/access.rb#L38-L40

Setting the value to `true` or `false` would both have the same effect of disabling RBAC today.

Reply:
@brahmanim @gmcculloug @syncrou see https://github.com/RedHatInsights/e2e-deploy/pull/429